File: DK05.py
import math

def so_chinh_phuong(number : int):
    if number < 0:
        return "NO"
    number_sqrt = int(math.sqrt(number))
    if pow(number_sqrt,2) == number:
        return "YES"
    return "NO"

print(so_chinh_phuong(int(input())))

File: DK05/DK05.cpp
#include<bits/stdc++.h>
using namespace std;

string so_chinh_phuong(int number) {
    if (number < 0) return "NO";
    int number_sqrt = (int)sqrt(number);
    if (pow(number_sqrt, 2) == number) {
        return "YES";
    }
    return "NO";
}

int main() {
    int n;
    cin >> n;
    cout << so_chinh_phuong(n) << endl;
    return 0;
}
